### ISSUE TITLE
Domain blacklisting

### ISSUE THREAD

**Issue by khionu** [jagrosh/Vortex#37]
It would be nice if we could blacklist URLs by Domain, ie porn sites

**Comment by jagrosh** [comment]
This would likely be a subset of #2

**Comment by khionu** [comment]
Agreed, still thought it was different enough where it would deserve its own issue. If you feel differently, understandable
